packages/react-app/.eslintrc.json

```diff
@@ -42,6 +42,7 @@
     "react/no-unused-prop-types": "warn",
     "jsx-props-no-spreading": "off",
     "react/jsx-props-no-spreading": "off",
+    "react/state-in-constructor": "off",
```

Comment:
Needed because some preset (i think airbnb) and the ErrorBoundary component dont have constructor to receive the props.